conan/cli/formatters/report/__init__.py

```diff
@@ -0,0 +1 @@
+from .diff import format_diff_html, format_diff_txt, format_diff_json
```

Comment:
We don't use relative imports, but always absolute ones

Reply:
Note that this comes from the graph formatters using this same approach, should we modify it in all the rest of the formatters?